docs/getconsolealias.md

```diff
@@ -5,6 +5,9 @@ author: bitcrazed
 ms.author: richturn
 ms.topic: article
 keywords: console, character mode applications, command line applications, terminal applications, console api
+f1_keywords:
+- WINCON/GetConsoleAlias
+- GetConsoleAlias
```

Comment:
This one has both an A and W variant. It also has the #ifdef UNICODE defined variant without either letter.

I mention this because in other documents changed in this PR, the A variant of an API like this was chosen and in other ones the W variant was chosen.

Should we be listing all agnostic macro variants? All A variants? All W variants? Or all of the above?

Reply:
(There are many more string-related functions in here but I'm not going to mark each one with a comment until we answer this.)